php/util.php

```diff
@@ -393,6 +393,32 @@ function getUploadsPath( $user = null )
 	return( getProfilePath($user).'/torrents' );
 }
 
+function getWebUIJsonFile()
+{
+	global $profilePath;
+	$ret.= isset($profilePath) ? $profilePath : '../share';
+	$ret = str_replace('..', '', $ret);
+	
+	if(is_null($user))
```

Comment:
How $user may be not null here?

Reply:
Actually this variable is undefined here. I'm just going to remove this if statement and set it's value.